### PR TITLE
Accounting for activity in the previous week and month to calculate wau and mau.

### PR DESCRIPTION
**Pull request by lucia-vargas-a**

### CODE REVIEW

**Review by kwindau — APPROVED** [review]

**Review by bochocki — APPROVED** [review]
💯

File: backfill/2024-04-26-active_users_aggregates/desktop_query.sql [hunk 1]
```diff
@@ -30,8 +31,32 @@ WITH baseline AS (
...
+    client_id,
+    (
+      LOGICAL_AND(uri_count > 0)
+      AND LOGICAL_AND(active_hours_sum > 0)
+      AND LOGICAL_AND(days_since_seen = 0)
+    ) AS is_dau
+  FROM baseline
+  WHERE COALESCE(isp, "") <> 'BrowserStack'
+    AND COALESCE(distribution_id, "") <> 'MozillaOnline'
+  GROUP BY
```
Comment (bochocki) [inline backfill/2024-04-26-active_users_aggregates/desktop_query.sql]: These should not be filtered out here; instead we should calculate DAU for the app_names "Firefox Desktop BrowserStack" and "Firefox Desktop MozillaOnline".

We should be calculating DAU/WAU/MAU for all products and segments, even when they don’t necessarily count towards the KPI (Klar and BrowserStack for example). That lets us track e.g. DAU for those segments for reporting purposes (e.g. “How did Mobile Browserstack DAU change in 2024 YTD”), and is consistent with how AUA currently works:

```sql
SELECT submission_date,
       SUM(dau) AS BrowserStack_DAU,
  FROM `moz-fx-data-shared-prod.telemetry.active_users_aggregates`
 WHERE app_name IN ('Fenix BrowserStack',  'Firefox iOS BrowserStack', 'Focus Android BrowserStack', 'Focus iOS BrowserStack')
   AND submission_date >= '2024-01-01'
 GROUP BY submission_date
 ORDER BY submission_date
```
Thanks @jmsilverman for catching this!

**Review by bochocki — CHANGES_REQUESTED** [review]
@jmsilverman Caught a logic issue; we shouldn't be filtering BrowserStack and MozillaOnline out of the desktop backfill.

File: backfill/2024-04-26-active_users_aggregates/desktop_query.sql (hunk 1 above)
Comment (lucia-vargas-a) [reply backfill/2024-04-26-active_users_aggregates/desktop_query.sql]: Thank you! updated.

**Review by bochocki — APPROVED** [review]

**Pull request merged**